extensions/typescript-language-features/src/typeScriptServiceClientHost.ts

```diff
@@ -150,6 +151,60 @@ export default class TypeScriptServiceClientHost extends Disposable {
 					standardFileExtensions: [],
 				}, onCompletionAccepted);
 			}
+			// TODO: Still probably isn't the right place for this
```

Comment:
Can this be registered as a language feature? That way it only gets activated when our other language features activate

Reply:
I did, but I'm not sure I did it correctly. I tried converting the showErrorMessage calls to logging, but I couldn't find logging in any other language features, so I deleted the non-error logs and kept the others as showErrorMessage.